### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by Vindexus**
Removed xcomponent docs. Updated text to nano. Updated the demo npm script. Added a demo to the dist folder for serving locally.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<html>
```
Comment (bluepnume) [inline dist/index.html]: Can you move this to /demo?

**Comment by bluepnume** [comment]
Thanks for all the fixes! Could you remove the changes to /dist for now? Prefer to keep `/dist` changes for release-time only to keep the history clean.

**Comment by Vindexus** [comment]
I'll try to get those changes in this week :)

File: dist/index.html (hunk 1 above)
Comment (Vindexus) [reply dist/index.html]: Do you want this in a /demo folder at root or in a folder at /dist/demo?

If `npm run demo` command is reading from a `/demo` folder it won't have access to the js files in dist. Maybe I could do a symlink?

**Pull request closed without merging**